### ISSUE TITLE
[Feature Request]: 

### ISSUE THREAD

**Issue by builtinhold** [FreeTubeApp/FreeTube#4276]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

Privacy Redirect and LibRedirect extensions don't support Safari. There is Privacy Redirect port for Safari in the appStore but it only seems to support URL redirection and not opening the current URL in an alternate app. 

### Proposed Solution

Would it be possible to register FreeTube as an alternate browser on macOS, so it would appear in Safari's "Develop>Open Page With" menu?
Even if it is not an automatic process, at least we could just open the current YouTube URL displayed in Safari (and blocked by adblockers for example) directly in FreeTube.

### Alternatives Considered

Cannot find any alternative to copy the YouTube url and past it in an already open FreeTube instance.

### Issue Labels

ease of use improvement

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of https://github.com/FreeTubeApp/FreeTube/issues/3257
